cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -1974,6 +1974,20 @@
             "token": {
               "type": "string",
               "description": "Required contract token obtained from https://ubuntu.com/advantage to attach."
+            },
+            "config": {
+              "type": "object",
+              "description": "Configuration settings or override Ubuntu Advantage config",
+              "properties": {
+                "http_proxy": {
+                  "type": "string",
+                  "description": "Ubuntu Advantage HTTP Proxy"
+                },
+                "https_proxy": {
+                  "type": "string",
+                  "description": "Ubuntu Advantage HTTPS Proxy"
+                }
```

Comment:
I'd like us to add all known proxy config options to this schema here as they seem the most likely customers would want to adapt and unlikely to change name in the near future: http_proxy, https_proxy,  apt_http_proxy, apt_https_proxy,  ua_apt_http_proxy ,  ua_apt_https_proxy , global_apt_http_proxy, global_apt_https_proxy

```suggestion
                },
                "global_apt_http_proxy": {
                  "type": "string",
                  "format": "uri",
                  "description": "HTTP Proxy URL used for all APT repositories on a system. Stored at ``/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy``"
               },
                "global_apt_https_proxy": {
                  "type": "string",
                  "format": "uri",
                  "description": "HTTPS Proxy URL used for all APT repositories on a system. Stored at ``/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy``"
               },
                "ua_apt_http_proxy": {
                  "type": "string",
                  "format": "uri",
                   "description": "HTTP Proxy URL used only for Ubuntu Advantage APT repositories. Stored at ``/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy``"
               },
                "ua_apt_https_proxy": {
                  "type": "string",
                  "format": "uri",
                  "description": "HTTP Proxy URL used only for Ubuntu Advantage APT repositories. Stored at ``/etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy``"               },
```